src/frontend/src/app/pages/user/bucket/bucket-form-page/bucket-form-page.component.ts

```diff
@@ -154,8 +154,11 @@ export class BucketFormPageComponent implements OnInit, IsDirty {
           type: 'checkbox',
           name: 'VersioningEnabled',
           label: TEXT('Enabled'),
-          hint: TEXT('Enable versioning for the objects in this bucket.'),
+          hint: TEXT(
+            'Enable versioning for the objects in this bucket. Can only be enabled while bucket creation.'
```

Comment:
"while bucket creation" sounds weird. s/while/during? 

Technically this isn't true. 
Right now you can go from 
- unversioned to versioned any time. 
- versioned -> unversioned is not supported in S3. 
- versioned -> suspended versioned is not supported by us.